Description: Changes to send service-unavailable for not handled iq as said in RFC 6120.
Index: source/org/jivesoftware/smack/util/PacketParserUtils.java
===================================================================
--- source/org/jivesoftware/smack/util/PacketParserUtils.java	(revision 12558)
+++ source/org/jivesoftware/smack/util/PacketParserUtils.java	(working copy)
@@ -333,7 +333,7 @@
                 iqPacket.setTo(from);
                 iqPacket.setFrom(to);
                 iqPacket.setType(IQ.Type.ERROR);
-                iqPacket.setError(new XMPPError(XMPPError.Condition.feature_not_implemented));
+                iqPacket.setError(new XMPPError(XMPPError.Condition.service_unavailable));
                 connection.sendPacket(iqPacket);
                 return null;
             }
